frontend/scripts/index.ie.ejs

```diff
@@ -33,19 +33,19 @@
     </div>
     <div class="row align-center">
       <div class="column small-12 medium-3">
-        <a class="browser-box">
+        <a class="browser-box" href="https://www.google.com/chrome/">
           <span class="browser-title">Chrome</span>
           <img class="browser-logo" src="/images/browsers/chrome.svg" />
         </a>
       </div>
       <div class="column small-12 medium-3">
-        <a class="browser-box">
+        <a class="browser-box" href="https://www.mozilla.org/en-US/firefox/new/">
           <span class="browser-title">Firefox</span>
           <img class="browser-logo" src="/images/browsers/firefox.svg" />
         </a>
       </div>
       <div class="column small-12 medium-3">
-        <a class="browser-box">
+        <a class="browser-box" href="https://www.microsoft.com/en-us/windows/microsoft-edge">
```

Comment:
Maybe we could use the language here, or will it be too much?

Reply:
I think this is good enough for now